src/file_operations.rs

```diff
@@ -86,6 +89,14 @@ impl FileOperations {
         self.operations.push(Op::MkDirAll(self.root.join(name)))
     }
 
+    pub fn link<P: AsRef<Path>, Q: AsRef<Path>>(&mut self, source: P, dest: Q) {
+        self.operations.push(Op::Link(source.as_ref().to_path_buf(), self.root.join(dest)));
+    }
+
+    pub fn remove<P: AsRef<Path>>(&mut self, file: P) {
+        self.operations.push(Op::Remove(self.root.join(file)));
```

Comment:
So this command is going to be for removing files from the hermit shell directory, not for removing the symlinks that hermit creates (which would be in the user's home directory).  I think both of those operations will probably be necessary at some point, and the name should probably reflect the usage of `self.root.join`.  `remove_from_root` maybe? That's kinda wordy...